<?xml version="1.0"?>
<launch>
    <!-- Set parameter untuk device name dan baud rate -->
    <param name="devicename" value="/dev/U2D2" />
    <param name="baudrate" value="1000000" />
    <param name="motor_ids" value="[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]" />
    <param name="motor_names" value="['FRcoxa', 'FRfemur', 'FRtibia', 'FLcoxa', 'FLfemur', 'FLtibia', 'BRcoxa', 'BRfemur', 'BRtibia', 'BLcoxa', 'BLfemur', 'BLtibia']" />

    <!-- Start dynamixel controller node -->
    <node name="airosa_controller" pkg="airosa_controller" type="controller.py" output="screen" />

</launch>

